packages/web-components/src/components/ic-badge/ic-badge.stories.mdx

```diff
@@ -1004,3 +1004,78 @@ import readme from "./readme.md";
     </div> `}
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  accessibleLabel: "notification",
+  customColor: null,
+  maxNumber: 1000,
+  position: "far",
+  size: "default",
+  textLabel: "1",
+  type: "text",
+  variant: "neutral",
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    parameters={{ loki: { skip: true } }}
+    argTypes={{
+      customColor: {
+        control: { type: "text" },
+      },
+      position: {
+        options: ["far", "near", "inline"],
+        control: {
+          type: "radio",
+        },
+      },
+      size: {
+        options: ["small", "default", "large"],
+        control: {
+          type: "radio",
+        },
+      },
+      type: {
+        options: ["text", "dot", "icon"],
+        control: {
+          type: "radio",
+        },
+      },
+      variant: {
+        options: [
+          "neutral",
+          "success",
+          "warning",
+          "error",
+          "info",
+          "light",
+          "custom",
+        ],
+        control: {
+          type: "select",
+        },
+      },
+    }}
+    name="Playground"
+  >
+    {(args) =>
+      html`<ic-button variant="secondary">
+        <ic-badge
+          slot="badge"
+          accessible-label=${args.accessibleLabel}
+          custom-color=${args.customColor}
+          max-number=${args.maxNumber}
+          position=${args.position}
+          size=${args.size}
+          text-label=${args.textLabel}
+          type=${args.type}
```

Comment:
When type is set to icon, no icon is actually shown. Please could you add some conditional rendering here